lambdas/indexer/app.py

```diff
@@ -169,7 +169,7 @@ def index(event: chalice.app.SQSEvent):
     for record in event:
         message = json.loads(record.body)
         attempts = record.to_dict()['attributes']['ApproximateReceiveCount']
-        log.info(f'Worker handling message {message}, attempt #{attempts} (approx).')
+        log.info('Worker handling message %s, attempt #%s (approx).', message, attempts)
```

Comment:
I thought log messages were the exception to using f-strings. I don't see it mentioned in the ticket description but if the policy regarding log messages is being changed with this ticket then the contributing guide will need to be updated also

CONTRIBUTING.rst
> We don't use f'' strings or string concatenation when interpolating dynamic values into log messages:

Reply:
My changes extensively _removed_ the use of f-strings from log messages, except for a few cases where it seemed to seriously harm readability. I certainly didn't _add_ any f-strings in log messages. I don't understand what feedback I'm supposed to take away from this comment.

Reply:
Yes sorry. I don't know how I got confused by this but I somehow thought the diff was telling me the reverse of what it actually is. Please ignore original comment.